modules/ROOT/pages/scheduler-xml-reference.adoc

```diff
@@ -9,6 +9,31 @@ A XML for the Scheduler has these elements:
 * `scheduling-strategy` block inside `scheduler`.
 * `fixed-frequency` or `cron` element inside the `scheduling-strategy` block.
 
+== Properties of Scheduler (`<scheduler>`)
+[%header,cols="34,33,33"]
+|===
+|Attribute |Description |Example
+|`disallowedConcurrentExecution`
+|Skips the next scheduled flow execution if the last one has not ended. Next attempt to execute will be after another frequency period. Default value is `true`.
```

Comment:
This wording is unclear to me. 
To "skip" something means to not do it at all, whereas I THINK you mean is something like ...

"Postpones the next scheduled flow execution for a specified frequency period, until the current flow execution ends. Default value is `true`.

QUESTION: Do we need to say something about what the frequency period is?

Reply:
(BTW, I don't edit code examples. I just look...sometimes...for opening and closing formatting....)